ios-target/ViewModel/Home/TargetFormViewModel.swift

```diff
@@ -77,9 +84,16 @@ class TargetFormViewModel {
                         radius: targetArea,
                         topicId: targetTopic)
     TargetAPI.createTarget(target,
-                           success: { [weak self] target in
+                           success: { [weak self] response in
```

Comment:
I think that you have to expect the Target, topicId and user already parsed here